lms/static/js/student_account/views/LoginView.js

```diff
@@ -19,12 +21,15 @@
                 formType: 'login',
                 requiredStr: '',
                 submitButton: '.js-login',
+                formSuccessTpl: formSuccessTpl,
+                formStatusTpl: formStatusTpl,
```

Comment:
Why set these as variables? They are already available globally within the view as they are passed in via RequireJS.